modules/spotify.py

```diff
@@ -149,5 +159,13 @@ def query(jenni, input):
     except KeyError:
         notify(jenni, input.nick, "Unknown response from API server")
 
+def artist_list(data):
+    if (len(data) > 1):
+        artists = '\x02, \x02'.join(data[:-1])
+        artists = artists + '\x02 and \x02' + data[-1]
```

Comment:
Right now locally I have written it so a string would be like the below example. It would be correct to use instead of the `%s` and relevant ways of formatting the string?

```python
"Artist: {0}{1}{0}".format("\x02", artist['name'])
```